torch/csrc/distributed/rpc/utils.h

```diff
@@ -125,6 +126,12 @@ TORCH_API std::vector<at::IValue> readWrappedPayload(
     std::vector<char>& payload,
     const rpc::Message& message);
 
+TORCH_API void populateRemoteProfiledEvents(
```

Comment:
let's add some comments to describe this func

Reply:
Added these in the update.